RELEASE/scripts/autoscend/combat/auto_combat.ash

```diff
@@ -81,6 +81,16 @@ string auto_combatHandler(int round, monster enemy, string text)
 	{
 		awol_combat_helper(text);
 	}
+	if(my_path() == "Pocket Familiars")
```

Comment:
Might want to use if(in_pokefam()) here rather than this, primarily for neatness/consistency?

Reply:
Thanks, I changed in new commit.  I was mimicking previous style since I can barely code anything